Applications/Utils/MeshEdit/moveMeshNodes.cpp

```diff
@@ -151,7 +134,7 @@ int main (int argc, char* argv[])
     // maps the elevation of mesh nodes according to a ground truth mesh whenever nodes exist within max_dist
     if (current_key == "-MESH")
     {
-        if (argc < 5)
+        if (argc < 4)
```

Comment:
For another PR; this cli arguments handling could be upgraded to tclap.

Reply:
I know. However, this would require more time than I can currently spend on this. It's on the to-do list.